Bottom: 7d421cd2ed62c327f5681827025326f8cff99301
Top:    9f8cb57f2d8d330d3ca86e64859a6e45e490e7fa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-10-27 16:57:14 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
